rules/no-unreadable-array-destructuring.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const isCommaFollwedWithComma = (el, index, array) => {
+	if (el === null) {
+		return array[index + 1] === el;
```

Comment:
I think it would be more explicit to change the condition to `array[index + 1] === null`